src/store/modules/user.js

```diff
@@ -12,7 +12,7 @@ export const state = {
 export const mutations = {
   SET_CURRENT_USER(state, currentUser) {
     // console.log(state, currentUser)
-    state.currentUser = currentUser
+    state.currentUser =currentUser
```

Comment:
Add spacing for consistency.

Reply:
Remember to look through all your changes before commit. That way you are less likely to get stuff like this
```suggestion
    state.currentUser =c urrentUser
```